server/config/topic_page/disasters/AK.textproto

```diff
@@ -0,0 +1,49 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+## This file specifies the topic page config for poverty in USA
```

Comment:
Update this comment? Is this file just for testing purposes because we probably want to add the histogram chart to the actual disaster dashboard config here: https://github.com/datacommonsorg/website/blob/master/server/config/disaster_dashboard/Earth.textproto

Reply:
Removed!